styleguide/source/assets/scss/04-templates/_three_column.scss

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.three-column {
+  display: grid;
+  display: -ms-grid;
+  grid-template-columns: 1fr 2fr  1fr;
+  grid-template-rows: auto auto auto;
+  min-height: 30em;
+  -ms-grid-columns: 1fr 2fr 1fr;
+  -ms-grid-rows: auto auto auto;
+}
+
+article {
+  grid-column: 1 / 5;
+  grid-row: 1;
+
+  @media (min-width: $bp-small) {
+    grid-column: 1 / 4;
+    grid-row: 1 / 4;
+  }
+
+  @media (min-width: $bp-med) {
+    grid-column: 2;
+    grid-row: 1 / 4;
+    -ms-grid-column: 2;
+    -ms-grid-row: 1;
+  }
+}// end article
+
+
+aside.ama__sidebar-primary {
```

Comment:
Was there any particular reasoning for including the HTML element in the styling? We should style based on the class only when possible.

Reply:
> By using html elements it's clear we are modifying the layout and not the children.

I might not understand this reasoning - when would the children have this class? Can you give an example? 

In general, if we want to establish this as a rule and it's not an intuitively obvious standard (which AFAIK it isn't, but I could be wrong), we should document it somewhere. If documenting it somewhere would add bloat, it might mean the rule is too specific or complicated to be added to our standards.

Reply:
I can go either way. Let's just stick to classes for now because that precedence was set in SG1